### PR TITLE
Update information.md

### PR DESCRIPTION
**Pull request by CrankyLandlord**
tried to make things more succinct. First draft so may not be perfect 🫡

### CODE REVIEW

File: Docs/topics/discord/information.md [hunk 1]
```diff
@@ -3,43 +3,31 @@ channelId: 1049224956162097212
...
+This list is non-exhaustive of behavior we might take action on, but in effect, use common sense and all will be well. If you have any questions about our rules feel free to contact a staff member; if you have suggestions you can create a GitHub issue on our [documentation repo.](<https://github.com/LittleBigRefresh/Docs/issues/new>)
 
 ## Enforcement
 
-We generally try to be hands-off when it comes to punishment. In most cases, the offending content will simply be deleted, or you will receive a verbal warning. In more extreme cases, you will be muted for anywhere between an hour and a week. This mostly happens after persisting and being told to stop doing something by a staff member.
+Generally, we try to be hands-off when it comes to punishment. Mistakes happen and everyone has off days–more often than not we might simply ask you to stop doing something and the offending content will be deleted. In more extreme cases you might receive a verbal warning from our bot–again this isn’t a punishment so to speak but something we issue in the hopes that behavior changes.
```
Comment (jvyden) [inline Docs/topics/discord/information.md]: > In more extreme cases you might receive a verbal warning from our bot

Our bot doesn't do this; (unless Dyno has been doing that without my knowledge?) all actions apart from DO are manual.

File: Docs/topics/discord/information.md [hunk 2]
```diff
@@ -3,43 +3,31 @@ channelId: 1049224956162097212
...
-
-Generally though, most punishments are given out for a reason. Due to our hands-off nature we tend to focus on general behaviour rather than individual incidents. For example, if your punishment was delivered as a result of bad behavior over the past month, we won't be accepting appeals regarding behavior over the past day. Some things may be what cause punishments, but more often than not they are not the whole story.
+We currently lack a proper appeals system due to lack of cases. However, if you believe a punishment was unfairly administered, you can reach out to a staff member–preferably not the one who issued the punishment (as to avoid bias)–and explain your perspective. Most punishments we issue tend to focus on general behavior rather than individual incidents so in most instances we probably won’t reverse a decision.
```
Comment (jvyden) [inline Docs/topics/discord/information.md]: You can also mention that I can be contacted at `support@littlebigrefresh.com` or `jvyden@jvyden.xyz` for appeals.

I think we should have a proper system for appeals by now but we can talk about that later.

File: Docs/topics/discord/information.md [hunk 3]
```diff
@@ -3,43 +3,31 @@ channelId: 1049224956162097212
...
+Generally, we try to be hands-off when it comes to punishment. Mistakes happen and everyone has off days–more often than not we might simply ask you to stop doing something and the offending content will be deleted. In more extreme cases you might receive a verbal warning from our bot–again this isn’t a punishment so to speak but something we issue in the hopes that behavior changes.
 
-More often than not, a compromise by a staff member is given. For example, one may be told to move to a different channel or thread. Not following this advice will almost definitely result in a mute.
-
-If you have proven to consistently be disrespectful to the community, only then will you be banned.
+With that being said, if you deliberately and/or repeatedly push buttons, we will not hesitate to mute or ban you depending on the severity of your offenses. Participation in our community is a privilege and we’re not obligated to keep providing service if you don’t respect that.
```
Comment (jvyden) [inline Docs/topics/discord/information.md]: Probably too harsh language

File: Docs/topics/discord/information.md [hunk 4]
```diff
@@ -3,43 +3,31 @@ channelId: 1049224956162097212
...
-3. Treat everyone with respect. Absolutely no harassment, witch hunting, sexism, racism, or hate speech will be tolerated.
-4. No age-restricted or obscene content. This includes text, images, or links featuring nudity, sex, hard violence, or other graphically disturbing content.
-5. Please keep criticism constructive. We generally won't take action for this, but it will be noticed if you are consistently negative when discussing topics with other members.
-6. Discussion of piracy won't be tolerated. LittleBigRefresh is not the place to discuss this type of thing. We won't mute you for simply mentioning piracy, but we definitely will take action if you're discussing methods to enable piracy, or suggesting that people commit piracy. Just keep it out of this server.
-7. Try to avoid discussing topics relating to politics, religion and world tragedies. Those topics can quickly cause discomfort and start arguments. Remember, this is a LittleBigPlanet server; we're here to discuss and play LittleBigPlanet!
+3. Treat everyone with respect. Harassment, witch hunting, rage baiting, sexism, racism, hate speech, any other form of hateful conduct or targeting, will not be tolerated.
```
Comment (jvyden) [inline Docs/topics/discord/information.md]:
```suggestion
3. Treat everyone with respect. Harassment, witch-hunting, rage-baiting, sexism, racism, hate speech, or any other form of hateful conduct or targeting will not be tolerated.
```

**Review by jvyden — CHANGES_REQUESTED** [review]
Looks pretty good to me, but there's a couple weird spots

File: Docs/topics/discord/information.md (hunk 3 above)
Comment (Beyley) [reply Docs/topics/discord/information.md]: agree, probably best if the secondhalf is just removed
```suggestion
With that being said, if you deliberately and/or repeatedly push buttons, we will not hesitate to mute or ban you depending on the severity of your offenses.
```

**Review by jvyden — APPROVED** [review]

**Review by Beyley — APPROVED** [review]

**Review by turecross321 — APPROVED** [review]
Looks good 👍

**Pull request merged**